### PR TITLE
Add norm eval in configs

### PR DESCRIPTION
**Pull request by xvjiarui**
This PR add `norm_eval= True` in all configs files.

### CODE REVIEW

**Comment by sudo-rm-covid19** [comment]
What is the reason for fixing the norms of backbones if pretrained weights are used during training?

**Comment by xvjiarui** [comment]
Hi @sudo-rm-covid19 
In resnet based model, batch size is usually small, so batch statistics is not accurate enough.

**Comment by sudo-rm-covid19** [comment]
> Hi @sudo-rm-covid19
> In resnet based model, batch size is usually small, so batch statistics is not accurate enough.
> So fixed BN is a common practice to address that.
Hi jiarui @xvjiarui ,
Thanks for your prompt reply. From my oberservation, if norm_eval sets True, the loss decreases at a lower rate. But if I set it False, the training error could diverge suddenly at some point (it might happen several epochs after the start of the training and every thing seems good in prior to that.)
Does it mean it will provide more stable training process but the results might be sub-optimal (in comparison to the scenario when we have larger batch size)?

**Comment by xvjiarui** [comment]
Hi @sudo-rm-covid19 
Yes. So we could better result by using SyncBN with `norm_eval=False`. But we need to use `4Conv1FcHead` instead of `2FCHead` for that.

**Comment by sudo-rm-covid19** [comment]
> Hi @sudo-rm-covid19
> Yes. So we could better result by using SyncBN with `norm_eval=False`. But we need to use `4Conv1FcHead` instead of `2FCHead` for that.

Hi @xvjiarui 
Thanks for your advice!
